src/DotNetWorker.Grpc/GrpcServiceCollectionExtensions.cs

```diff
@@ -68,9 +68,14 @@ public static IServiceCollection AddGrpc(this IServiceCollection services)
 #endif
 
             services.AddOptions<GrpcWorkerStartupOptions>()
-                .Configure<IConfiguration>((arguments, config) =>
+                .Configure<IConfiguration>((grpcWorkerStartupOption, config) =>
                 {
-                    config.Bind(arguments);
+                    grpcWorkerStartupOption.Host = config["host"];
+                    grpcWorkerStartupOption.Port = config.GetValue<int>("port");
+                    grpcWorkerStartupOption.Uri = config["functions-uri"];
+                    grpcWorkerStartupOption.RequestId = config["functions-requestid"] ?? config["requestId"];
```

Comment:
An alternative is to use `switchMappings` when adding command line args. This way we can map `--functions-requestid` to be `RequestId` in `IConfiguration`, and so on.

https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Configuration.CommandLine/src/CommandLineConfigurationExtensions.cs#L155